baseapp/abci.go

```diff
@@ -178,7 +188,7 @@ func (app *BaseApp) Query(_ context.Context, req *abci.RequestQuery) (resp *abci
 	// handle gRPC routes first rather than calling splitPath because '/' characters
 	// are used as part of gRPC paths
 	if grpcHandler := app.grpcQueryRouter.Route(req.Path); grpcHandler != nil {
-		return app.handleQueryGRPC(grpcHandler, req), nil
+		return app.handleQueryGRPC(grpcHandler, req, ctx), nil
```

Comment:
makes more sense to have the ctx be first argument here

Reply:
fixed [put ctx argument first](https://github.com/cosmos/cosmos-sdk/pull/25516/commits/faf5c1d05c66548671baf887de4e7c8f97878e4f)